README.md

```diff
@@ -40,15 +33,19 @@ aws:
   instance:
     count: 5
 ```
-5. Build the plugin
+3. Build the plugin
 ```
 $ mkdir plugins
```

Comment:
Looks like this is rendered moot by the addition of the file `plugins/attach.pp`?

Reply:
Fixed